packages/amplify-cli/src/extensions/amplify-helpers/get-env-info.js

```diff
@@ -7,6 +7,9 @@ function getEnvInfo() {
   let envInfo = {};
   if (fs.existsSync(envFilePath)) {
     envInfo = readJsonFile(envFilePath);
```

Comment:
We can use the getEnvInfo() out here right as well right?

Reply:
What do you mean, this IS ```getEnvInfo``` itself.